pkg/webhooks/validation/controller/controller.go

```diff
@@ -107,124 +104,70 @@ func NewValidatingWebhookController(client kube.Client,
 		CABundleWatcher:  caBundleWatcher,
 		Revision:         revision,
 		ServiceName:      "istiod",
+		Backoff:          time.Second,
+		Retries:          5,
 	}
 	return newController(o, client)
 }
 
-type eventType string
-
-const (
-	retryEvent  eventType = "retryEvent"
-	updateEvent eventType = "updateEvent"
-)
+func newController(o Options, client kube.Client) *Controller {
+	c := &Controller{
+		o:      o,
+		client: client,
+	}
 
-type reconcileRequest struct {
-	event       eventType
-	description string
-	webhookName string // if empty, ALL webhooks should be updated in response
-}
+	c.queue = controllers.NewQueue("validation",
+		controllers.WithReconciler(c.Reconcile),
+		controllers.WithMaxAttempts(o.Retries),
+		controllers.WithRateLimiter(workqueue.NewItemExponentialFailureRateLimiter(o.Backoff, 5*time.Minute)))
```

Comment:
```
		// Try first few(5) retries quickly so that we can detect true conflicts by multiple Istiod instances fast.
		// If there is a conflict beyond this, it means Istiods are seeing different ca certs and are in inconsistent
		// state for longer duration. Slowdown the retries, so that we do not overload kube api server and etcd.
		controllers.WithRateLimiter(workqueue.NewItemFastSlowRateLimiter(100*time.Millisecond, 1*time.Minute, 5)),
```


long time we may need to unify webhook patch for inject and validation